docs/guide/adapters/README.md

```diff
@@ -145,6 +145,30 @@ _(in order of first appearance)_
     ![](../../images/jetstick_z2.png)
     </details>
 
+  * <details>
+    <summary>SMLIGHT SLZB-07p7 CC2652P7 Zigbee USB adapter - the first CC2652P7 USB adapter</summary>  
+
+    Powerful, tiny modern design, developed for Zigbee2MQTT, pre-flashed and ready to use Zigbee adapter. Autodiscovery in Home Assistant. AutoBSL (remote update) enabled.  
+    Rich packing that includes:
+    | Package includes | SLZB-07p7 adapter | +3dB antenna 360° | QR-manual |
+    |:-|:-:|:-:|:-:|
+    |Quantity: |1 pcs|1 pcs|1 pcs|1 pcs|  
+    |Image| [![](https://smlight.tech/external-content/images/zigbee2mqtt-io/slzb-07p7/slzb-07p7-adapter-small.jpg)](https://smlight.tech/external-content/images/zigbee2mqtt-io/slzb-07p7/slzb-07p7-adapter.jpg) | [![](https://smlight.tech/external-content/images/zigbee2mqtt-io/slzb-07p7/slzb-07x-antenna-small.jpg)](https://smlight.tech/external-content/images/zigbee2mqtt-io/slzb-07p7/slzb-07x-antenna.jpg) | [![](https://smlight.tech/external-content/images/zigbee2mqtt-io/slzb-07p7/slzb-07x-qr-small.jpg)](https://smlight.tech/external-content/images/zigbee2mqtt-io/slzb-07p7/slzb-07x-qr.jpg) |  
+
+    Remote Zigbee update, autodiscovery, LEDs, and flashing pins. It fits home and business use.  
+    
+    - Zigbee firmware:  
+    -- [Firmwares folder](https://github.com/Koenkk/Z-Stack-firmware/tree/master/coordinator/Z-Stack_3.x.0/bin) for manual updates - use CC2652P7 firmwares.    
```

Comment:
Please link to coordinator/router firmware like done for all other CC1352/CC2652 devices.

Reply:
> Please link to coordinator/router firmware like done for all other CC1352/CC2652 devices.

Done (just in case, no router firmware for CC2652P7 chip). However, I think it is better to have a link to the folder rather than a direct link to the specific version. Otherwise, users do not see updates. But I did as you suggested, a direct link to the fw.

Reply:
Please do it exactly the same as the others, I will take care of updating the links when there is a new fw

Reply:
> Please do it exactly the same as the others, I will take care of updating the links when there is a new fw

done https://github.com/Koenkk/zigbee2mqtt.io/pull/2718/commits/837a91965a57f4151b635eea566e6a2b8a987c3b